packages/cli/src/run.test.ts

```diff
@@ -49,3 +44,64 @@ describe("Run command", () => {
     }
   );
 });
+
+describe("Fix command", () => {
+  it.each([
+    ["package-one", "fix", "lf", 0],
+    ["package-one", "fix", "crlf", 0],
+  ] as const satisfies [string, string, LineEndings, number][])(
+    'should execute "%s %s" without changing line endings from %s',
+    async (arg0, arg1, sourceLineEnding, expectedExitCode) => {
+      // arrange
+      const temp = f.copy("basic-with-scripts");
+      const filePath = path.join(temp, "package.json");
+      convertFileLineEndings(filePath, sourceLineEnding);
+      // act
+      const { exitCode } = await executeBin(temp, "fix", arg0, arg1);
+      // assert
+      expect(exitCode).toBe(expectedExitCode);
+      const fixedPackageFile = fs.readFileSync(filePath, "utf8");
+      f.cleanup();
+      expect(detectLineEndings(fixedPackageFile)).toBe(sourceLineEnding);
+    }
+  );
+});
+
+type LineEndings = "crlf" | "lf";
+
+function convertFileLineEndings(path: string, targetLineEnding: LineEndings) {
+  let file = fs.readFileSync(path, "utf8");
+  // detect mixed line endings
+  if (
+    file.includes("\r\n") &&
+    (file.match(/\r\n/g) || []).length !== (file.match(/\n/g) || []).length
+  ) {
+    throw new Error("mixed line endings in fixture file: " + path);
+  }
```

Comment:
I'd print a warning or something but format with `\n`. Throwing could be considered a breaking change.

Reply:
ah, this is just a test code - this is fine